packages/frontend/src/components/ProjectAccess/ProjectAccess.tsx

```diff
@@ -57,50 +65,78 @@ const UserListItem: FC<{
             <ItemContent>
                 <SectionWrapper>
                     <UserInfo>
-                        <UserName className={Classes.TEXT_OVERFLOW_ELLIPSIS}>
+                        <UserName>
                             {firstName} {lastName}
                         </UserName>
-                        {email && <UserEmail minimal>{email}</UserEmail>}
+
+                        {email && <EmailBox>{email}</EmailBox>}
                     </UserInfo>
 
                     {relevantOrgRole && (
-                        <Tooltip2
-                            content={`This user inherits the organization role: ${relevantOrgRole}`}
+                        <Tooltip
+                            position="left"
+                            color="dark"
+                            withArrow
+                            arrowSize={10}
+                            label={`This user inherits the organization role: ${relevantOrgRole}`}
+                            sx={{ padding: '10px', backgroundColor: '#57606a' }}
```

Comment:
We're aiming to avoid overrides of the default component colors. Is there a specific reason for this override?

Reply:
Hello @IrakliJani,
by default the tooltip from mantile looks like this


![Screenshot (99)](https://github.com/lightdash/lightdash/assets/86042508/01bdd54d-cf86-426d-b984-f622d99da154)


and in Lightdash it looks like this


![Screenshot (100)](https://github.com/lightdash/lightdash/assets/86042508/f8a9448e-bebc-49f4-a974-1a8552478af6)

If you are OK with the by-default looks of mantile, I can change that accordingly